db/migrate/20160712043931_create_storage_profiles.rb

```diff
@@ -0,0 +1,11 @@
+class CreateStorageProfiles < ActiveRecord::Migration[5.0]
+  def change
+    create_table :storage_profiles do |t|
+      t.string :name
+      t.string :uuid
+      t.string :profile_type
```

Comment:
That's right. As we've discussed, probably the name is good enough for what we are targeting at this stage. But figure the profile type may be useful to be included.